dev/package-examples/nginx-1.2.0/input/access/agent/input/input.yml

```diff
@@ -0,0 +1,14 @@
+# This is not an array on purpose to make sure only 1 single input is specified in this file.
```

Comment:
In a meeting, @skh brought up a good point here that we have a double meaning of input here (shows up twice in the path). We didn't reach a conclusion on what this should be called, but `dataset` was mentioned as an option.

@exekias @jsoriano @ph Some thoughts on this?

Reply:
I have been thinking about this a bit more. A dataset is basically a template for an input with all its assets. All inputs with the data set `access.log` look exactly the same in the end. An input can exist multiple times, still it is the same dataset. So I think this fits well here. Will rename.

Reply:
Just pushed a commit with renaming it. One thing I realised is that the agent now does not have an input anymore, but only streams. So the above was renamed to `agent/stream/config.yml`. @ph Does this sound correct?